packages/react/src/components/pickers/BasePicker.tsx

```diff
@@ -883,23 +885,29 @@ export class BasePicker<T, P extends IBasePickerProps<T>>
     return currentIndex > -1 && !this.state.suggestionsLoading ? 'sug-' + currentIndex : undefined;
   }
 
-  protected getSuggestionsAlert(suggestionAlertClassName: string = legacyStyles.screenReaderOnly) {
```

Comment:
With the way BasePicker is set up, this is essentially a public API. So it might be better to either keep the existing name, or add a new method with the implementation and mark this one as deprecated.

Reply:
I'll add it back and mark it as deprecated, since the new one is different enough 👍